server/src/constants.js

```diff
@@ -1,3 +1,6 @@
-const distantFuture = 2147483647;
+export const distantFuture = 2147483647;
 
-export default distantFuture;
+export const nowInUTC = () => {
```

Comment:
this really doesn't belong in `constants.js` you could make this `time.js` or `util.js`...